src/Xamarin.Android.Build.Tasks/Xamarin.Android.Build.Tasks.csproj

```diff
@@ -19,7 +19,7 @@
     <DebugType>full</DebugType>
     <Optimize>False</Optimize>
     <OutputPath>..\..\bin\$(Configuration)\lib\xbuild\Xamarin\Android</OutputPath>
-    <DefineConstants Condition="'$(MonoDroidTiming)' == ''">TRACE;DEBUG;HAVE_CECIL;MSBUILD</DefineConstants>
+    <DefineConstants Condition="'$(MonoDroidTiming)' == ''">TRACE;DEBUG;HAVE_CECIL;MSBUILD;ANDROID_24</DefineConstants>
```

Comment:
Good question. It doesn't matter. Why? We had some awkward intermediate fields for any new API dependencies so that the new members were not referenced via the actual attribute properties.
https://github.com/xamarin/monodroid/blob/master/tools/msbuild/Mono.Android/ActivityAttribute.Partial.cs#L19

The hack did not last here because the entire attribute type is new in API Level 24.